Sources/Clappr_iOS/Classes/Base/Player.swift

```diff
@@ -87,7 +92,8 @@ open class Player: BaseObject {
         bindMediaControlEvents()
         bindPlaybackEvents()
         bindNotifications()
-
+        core?.updateChromelessMode(with: options)
+        updateChromelessModeVisibility()
```

Comment:
If we move the `updateChromelessModeVisibility()` to `Core` as a `didSet` callback for example, there's actually no need for these methods to exist and be called during `init` or `configure`.

Reply:
Actually on the init the `options` didSet is not called. 
Reference:  Swift Documentation 
[_willSet and didSet observers are not called when a property is first initialized. They are only called when the property’s value is set outside of an initialization context._](https://docs.swift.org/swift-book/LanguageGuide/Properties.html#//apple_ref/doc/uid/TP40014097-CH14-XID_368)

Reply:
Completely forgot that. Thanks 😉